### ISSUE TITLE
show if SSH is enabled on NSX Edge

### ISSUE THREAD

**Issue by michaelshaoul** [vmware/powernsx#593]
I would like a powershell / powercli or powernsx command to show if SSH is enabled on my NSX Edge appliances.

or an api to do the same. (This below is for the NSX Manager appliance. )
((Invoke-NsxRestMethod -uri "/api/1.0/appliance-management/components/component/ssh"-method "get").component

componentId         : SSH
name                : SSH Service
description         : Secure Shell
status              : RUNNING
enabled             : true
showTechSupportLogs : false
usedBy              :
componentGroup      : SYSTEM

**Comment by michaelshaoul** [comment]
Get-NsxEdge  | Disable-NsxEdgeSsh
could also add a confirm -eq false

**Comment by alagoutte** [comment]
if you don't specify name for _Get-NsxEdge_ all edge will be changed

for looking status, you need to look cliSettings on Get-NsxEdge

```powershell
(Get-nsxEdge edge01).cliSettings                                 

remoteAccess userName sshLoginBannerText
------------ -------- ------------------
                                                                                      
false        admin    ...         
```                      
and it is the **remoteAccess** field

a Get-NsxCliSettings will be available when #439 will be merged

**Comment by michaelshaoul** [comment]
thanks, that worked perfectly
